.circleci/config.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       image: ubuntu-2004:202111-01
       docker_layer_caching: true
 
-    working_directory: ~/kinto-dist
+    working_directory: ~/remote-settings
```

Comment:
Are we safe to remove this `working_directory` key here and below? I'm not sure what this does for us vs using the default `working_directory` path.